<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formFifo</class>
 <widget class="QWidget" name="formFifo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fifo</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblImage">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>barbu.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblWorkingDirectory">
       <property name="text">
        <string>1 - Select Working directory</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="btnWorkingDirectory">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Working directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblProject">
       <property name="text">
        <string>2 - Select Project</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbProject">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Project&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblSchema">
       <property name="text">
        <string>3 - Select Main schema</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbSchema">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Main schema&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblQuartus">
       <property name="text">
        <string>4 - Select Path for quartus</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="btnPathQuartus">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Path for quartus&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>5 - Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenerateProjectFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generate project file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>6 - Generate project file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenerateBitstream">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generate bitstream&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>7 - Generate bitstream</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuartusPgmw">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>8 - Quartus_pgmw</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_all_actions">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>140</height>
        </size>
       </property>
       <property name="text">
        <string>All actions</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtOutput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
